### PR TITLE
Fix showing AM/PM in post time when in 12-hours clock mode

### PR DESCRIPTION
**Pull request by koxen**
#### Summary
Quickfix for what I broke with my recent commit and got unnoticed by reviewers.

#### Ticket Link
[Please link the GitHub issue or Jira ticket this PR addresses.]

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] Ran `make check-style` to check for style errors (required for all pull requests)
- [ ] Ran `make test` to ensure unit and component tests passed
- [ ] Added or updated unit tests (required for all new features)
- [ ] Has server changes (please link)
- [ ] Has redux changes (please link)
- [ ] Has UI changes
- [ ] Includes text changes and localization file ([.../i18n/en.json](https://github.com/mattermost/mattermost-webapp/blob/master/i18n/en.json)) updates
- [ ] Touches critical sections of the codebase (auth, posting, etc.)

### CODE REVIEW

File: components/post_view/post_time.jsx [hunk 1]
```diff
@@ -47,18 +47,24 @@ export default class PostTime extends React.PureComponent {
...
+        const militaryTime = this.props.useMilitaryTime;
 
-        const hour = this.props.useMilitaryTime ? date.getHours() : (date.getHours() % 12 || 12);
+        const hour = militaryTime ? date.getHours() : (date.getHours() % 12 || 12);
         let minute = date.getMinutes();
         minute = minute >= 10 ? minute : ('0' + minute);
+        let time = '';
+
+        if (militaryTime) {
+            time = (hour >= 12 ? ' PM' : ' AM');
```
Comment (ccbrown) [inline components/post_view/post_time.jsx]: I assume the intention is to show AM/PM for civilian time, right? Not military time. If so, this is totally off.

Comment (koxen) [reply components/post_view/post_time.jsx]: oops you are right...updated 😅

File: components/post_view/post_time.jsx [hunk 2]
```diff
@@ -47,18 +47,24 @@ export default class PostTime extends React.PureComponent {
...
+        const militaryTime = this.props.useMilitaryTime;
 
-        const hour = this.props.useMilitaryTime ? date.getHours() : (date.getHours() % 12 || 12);
+        const hour = militaryTime ? date.getHours() : (date.getHours() % 12 || 12);
         let minute = date.getMinutes();
         minute = minute >= 10 ? minute : ('0' + minute);
+        let time = '';
+
+        if (!militaryTime) {
+            time = (hour >= 12 ? ' PM' : ' AM');
```
Comment (ccbrown) [inline components/post_view/post_time.jsx]: For civilian time, `hour` is always 1-12 at this point. 'AM' will be displayed for 22 hours a day.

**Comment by hmhealey** [comment]
Looks good. Thanks for the quick followup fix

**Pull request merged**